pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -181,8 +183,10 @@ def test_all(self):
         self.addCleanup(client.delete, repo['_href'])
 
         # Create a remote with the unsigned RPM fixture url.
-        # We need to use the unsigned fixture because the one used down below has unsigned RPMs.
-        # Signed and unsigned units have different hashes, so they're seen as different units.
+        # We need to use the unsigned fixture because the one used down below
```

Comment:
I assume you want to stick with 80 character line lengths then?

With this one comment, it would be better to re-wrap it so that ```Signed and``` begins where the other sentence ends.

Reply:
Updated.